electron_app/src/electron-main.js

```diff
@@ -45,47 +46,6 @@ const INITIAL_UPDATE_DELAY_MS = 30 * 1000;
 let mainWindow = null;
 let appQuitting = false;
 
-function safeOpenURL(target) {
```

Comment:
#4021 moved it to another file, #3993 brought it back (duplicated) because of a bad merge
this removes it once again, it is in the other file now.